backend/src/v2/compiler/argocompiler/argo.go

```diff
@@ -43,9 +45,18 @@ type Options struct {
 	PipelineRoot string
 	// optional
 	CacheDisabled bool
+	// optional
+	DefaultWorkspace *k8score.PersistentVolumeClaimSpec
+	// optional, default workspace size from API server config
+	DefaultWorkspaceSize string
 	// TODO(Bobgy): add an option -- dev mode, ImagePullPolicy should only be Always in dev mode.
 }
 
+const (
+	fallbackWorkspaceSize = "10Gi"
```

Comment:
to clarify, this is if no size is set in apiserverconfig but we specify workspace in apiserver config right? if so maybe a inline comment here would help clarify this for future devs